crates/rooch-integration-test-runner/src/lib.rs

```diff
@@ -142,8 +145,28 @@ impl<'a> MoveOSTestAdapter<'a> for MoveOSTestRunner<'a> {
                     let (_, module_id) = m.module_id();
                     let mut bytes = vec![];
                     m.named_module.module.serialize(&mut bytes).unwrap();
-                    let op = Op::New(bytes);
-                    change_set.add_module_op(module_id, op).unwrap();
+                    let mut entries = BTreeMap::new();
+                    entries.insert(
```

Comment:
The StateChangeSet also needs to add some functions like `add_module_op` to operate the changeset easily.